frontend/src/components/home/midsection/mid.css

```diff
@@ -31,14 +31,15 @@
 }
 
 .upper-mid-sectiontext {
+  text-align: center;
   font-size: 15pt;
   color: #697689;
 }
 
 .sub-heading {
```

Comment:
this class is used also in the about page so i think we just want this change for the Landing page so lets add an extra class name to handle that and put the different padding in there